packages/airnode-deployer/src/entrypoint.ts

```diff
@@ -18,12 +18,12 @@ export async function run(...args: unknown[]) {
   return handler.run(...args);
 }
 
-export async function processHttpRequest(...args: unknown[]) {
+export async function handleHttpReq(...args: unknown[]) {
```

Comment:
Do we want to add a comment explaining why is the fn name abbreviated? We are consistent in naming in Airnode and this is tempting to rename...

Reply:
Nit: Also maybe better name would be `httpRequest` (dropping the "handle" part)

Reply:
Actually, after thinking about this more I dislike that the "function name length" pollutes the deployer handlers... I'd propose the following. Create a file `entrypoint` which will be called by TF and rename the current `index` file to `implementation`.

The entrypoint file should be documented that it exist because the function name length is restricted, but the implementation can be named freely.

Reply:
Sorry, I don't follow. But I can rename the functions just here, in `entrypoint.ts`, and leave the handlers (those in `aws` and `gcp` directories) as they were. But that feels even less consistent.

Reply:
Sorry, I missed that we already have the `entrypoint.ts` file. I'd change the function names in this file and document that this is needed because of GCP/AWS.

This allows us to deal with that problem only in entrypoint and use whatever name we want in the rest of the codebase.